cpp/ql/test/library-tests/dataflow/taint-tests/taint.cpp

```diff
@@ -1,5 +1,5 @@
 int source();
-void sink(...);
+void sink(...) {};
```

Comment:
This will make the argument to `sink` be considered non-escaping, which should help with the last test in the file once we get interprocedural buffer flow.